desktop/libs/indexer/src/indexer/indexers/sql.py

```diff
@@ -355,23 +357,30 @@ def create_table_from_local_file(self, source, destination, start_time=-1):
           _csv_rows.append(tuple(row))
 
         if _csv_rows:
-          csv_rows = str(_csv_rows)[1:-1]
 
-          if dialect in ('hive', 'mysql'):
-            sql += '''\nINSERT INTO %(database)s.%(table_name)s VALUES %(csv_rows)s;\n'''% {
+          insert_sql = ""
+          for count in range(1 + len(_csv_rows)//5000):
```

Comment:
Nit: is this 5000 some limit? Can we comment about it or assign to a variable and use?